.github/workflows/sync-and-rebase-pr-from-fork.yml

```diff
@@ -42,6 +42,14 @@ jobs:
           git remote add contributor "https://github.com/$headRepositoryOwnerLogin/$headRepositoryName.git" || :
           contribHeadRefName="contributor-$headRepositoryOwnerLogin-$headRefName"
           git fetch --no-tags contributor +"$headRefName":"$contribHeadRefName"
+
+          # Rebase the branch with the latest master HEAD
+          git fetch origin master
```

Comment:
Do we care about contributor PRs that are not targeting master?

Reply:
Interesting! If that's the case, can we get that as an input along with the PR number (by default, it will be master)?

Reply:
Cool! Then I think `baseRefName=$($JQ ".baseRefName" <<<"$gh_pr_view")` will already have that value.

Reply:
Done!